cli.js

```diff
@@ -93,11 +94,22 @@ if (cli.flags.init) {
 	return;
 }
 
+var nodePaths;
+if (process.env.NODE_PATH) {
+    var osSplitChar = process.platform === 'win32' ? ';' : ':';
```

Comment:
Instead you can use `path.delimiter`. https://nodejs.org/api/path.html#path_path_delimiter